packit/api.py

```diff
@@ -452,15 +456,28 @@ def create_srpm(
         """
         self.up.run_action(actions=ActionName.post_upstream_clone)
 
-        source_dir = self.up.prepare_upstream_for_srpm_creation(
-            upstream_ref=upstream_ref
-        )
+        try:
+            source_dir = self.up.prepare_upstream_for_srpm_creation(
+                upstream_ref=upstream_ref
+            )
+        except Exception as ex:
+            raise PackitSRPMException(
+                f"Preparing of the upstream to the SRPM build failed: {ex}", ex
+            )
+
+        try:
+            srpm_path = self.up.create_srpm(
+                srpm_path=output_file, srpm_dir=srpm_dir, source_dir=source_dir
+            )
+        except PackitSRPMException as ex:
+            raise ex
```

Comment:
I think just `raise` should be enough here

Reply:
fixed